Camara Brasileira do Livro ISBN.js

```diff
@@ -0,0 +1,760 @@
+				"language": "pt-BR",
+				"libraryCatalog": "Câmara Brasileira do Livro ISBN",
+				"numPages": "1048",
+				"place": "São Paulo",
+				"publisher": "Editora 34",
+				"attachments": [],
+				"tags": [
+					{
+						"tag": "Literatura grega"
+					},
+					{
+						"tag": "Literatura."
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "search",
+		"input": {
+			"ISBN": "9786556752631"
+		},
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Manual de Sentença Trabalhista: Compreendendo a técnica da setença trabalhista para concurso",
+				"creators": [
+					{
+						"firstName": "Aline",
+						"lastName": "Leporaci",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Adriana Leandro de Sousa",
+						"lastName": "Freitas",
+						"creatorType": "contributor"
+					}
+				],
+				"date": "2023-03-02",
+				"ISBN": "9786556752631",
+				"abstractNote": "Nesse livro sobre sentença trabalhista, fase tão concorrida do concurso para a Magistratura do Trabalho, procuramos trazer os aspectos mais importantes a serem observados pelo candidato. O leitor poderá verificar a ordem de julgamento a seguir e a importância da fixação da prejudicialidade entre as matérias a serem analisadas. Além disso, também aprenderá as técnicas de distribuição do ônus da prova, e suas diversas teorias, sempre ressaltando qual deva ser de aplicação preferencial pelo candidato. O livro traz diversos aspectos teóricos, que são essenciais para a preparação de todos os interessados em efetivamente aprender a técnica da elaboração da sentença trabalhista, sempre com leitura fácil e direta. E não nos esquecemos dos aspectos práticos, pois o leitor terá exercícios de fixação de jornada de trabalho, e sentenças inéditas elaboradas pelas Autoras, com os respectivos gabaritos e sugestão de redação.",
+				"edition": "2",
+				"language": "pt-BR",
+				"libraryCatalog": "Câmara Brasileira do Livro ISBN",
+				"numPages": "232",
+				"place": "Rio de Janeiro",
+				"publisher": "FREITAS BASTOS EDITORA",
+				"shortTitle": "Manual de Sentença Trabalhista",
+				"attachments": [],
+				"tags": [
+					{
+						"tag": "Direito"
+					},
+					{
+						"tag": "Elaboração"
+					},
+					{
+						"tag": "Jornada"
+					},
+					{
+						"tag": "Magistratura"
+					},
+					{
+						"tag": "Trabalhista"
+					},
+					{
+						"tag": "Trabalho"
+					},
+					{
+						"tag": "Técnica"
+					},
+					{
+						"tag": "concurso"
+					},
+					{
+						"tag": "juiz"
+					},
+					{
+						"tag": "modelos"
+					},
+					{
+						"tag": "sentença"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "search",
+		"input": {
+			"ISBN": "9786559602513"
+		},
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Batman",
+				"creators": [
+					{
+						"firstName": "John",
+						"lastName": "Ridley",
+						"creatorType": "author"
+					},
+					{
```

Comment:
Worthwile adding additional handling for at least Jr., III, IV, and V?

Reply:
the portuguese standard is the first, `Moreira Filho, Álvaro Antônio Maia`, if the author itself doesnt decide on something else (which can happen)